rs/execution_environment/src/canister_manager/tests.rs

```diff
@@ -7081,6 +7081,141 @@ fn test_environment_variables_are_not_set_duplicate_keys() {
     );
 }
 
+// Helper function that updates the environment variables of a canister.
+fn update_settings_with_environment_variables(
+    test: &mut ExecutionTest,
+    canister_id: CanisterId,
+    env_vars: Vec<EnvironmentVariable>,
+) {
+    let args = UpdateSettingsArgs {
+        canister_id: canister_id.get(),
+        settings: CanisterSettingsArgsBuilder::new()
+            .with_environment_variables(env_vars)
+            .build(),
+        sender_canister_version: None,
+    };
+    test.subnet_message(Method::UpdateSettings, args.encode())
+        .unwrap();
+}
+
+// Helper function that fetches the environment variables from the canister status
+// and verifies they match the expected values.
+fn check_environment_variables_via_canister_status(
+    test: &mut ExecutionTest,
+    canister_id: CanisterId,
+    expected_env_vars: Vec<EnvironmentVariable>,
+) -> BTreeMap<String, String> {
```

Comment:
There's no need to return any value from this function since this value is guaranteed to be equal to `expected_env_vars`.